lib/ansible/modules/windows/win_find.ps1

```diff
@@ -172,6 +173,31 @@ Function Assert-Pattern($info) {
     $valid_match
 }
 
+Function Assert-Contains($info) {
+    $valid_match = $false
+
+    if ($contains -ne $null) {
+        $file_contents = [System.IO.File]::ReadAllText($info.path)
+        foreach ($pattern in $contains) {
+            if ($use_regex -eq $true) {
```

Comment:
Using the same parameter `$use_regex` for matching files and content of the files will lead to "collisions". What do you think about adding an additional parameter e.g. `contains_use_regex`?

Reply:
Another aspect of adding additional parameters. The win_find will have different parameters than the [`find`](https://docs.ansible.com/ansible/latest/modules/find_module.html#find-module) module.

Reply:
Having an additional parameter is a good idea. I don't think that's the end of the world differing module options between win_find and find. As long as we name it correctly we should be good.